src/contrib/cluster/Akka.Cluster.Sharding/ShardRegion.cs

```diff
@@ -553,16 +560,28 @@ private void ReplyToRegionStatsQuery(IActorRef sender)
             return Task.WhenAll(tasks);
         }
 
+        private List<ActorSelection> GracefulShutdownCoordinatorSelections
```

Comment:
I don't think, using property for work performing some computation + allocations is good choice. Either we should make it lazy/memoized if possible or turn into method if not.